### ISSUE TITLE
Invisible text on signup/login fields

### ISSUE THREAD

**Issue by appurist** [feathersjs/feathers-chat#34]
I seem to have invisible text on the `login.html` and `signup.html` pages in the feathers-chat example from the docs guide. Even the placeholder text is invisible.  

I've pulled the latest git and see the same thing.  It does it on the signup form and the login form but _not_ on the actual chat input field.

Also, I only seem to have this problem when using Firefox. It's fine under Chrome.  My version of Firefox is the latest at the time of filing this (49.0.1). 

It might be a problem with white on white text or something like that, but it looks fine in Chrome. It's just Firefox.

**Comment by jimtarber** [comment]
I have tracked the problem down to the padding style in `chat.css`:
```
main.login fieldset input,
main.signup fieldset input{
  width: 100%;
  padding: 1.4em 0.7em;
}
```
If you comment out the padding here, all is good (even in Firefox).

So this is why it only seems to affect those two pages.

**Comment by daffl** [comment]
Did we land a fix for this? I noticed this as well but can't remember is somebody submitted a PR with the changes already (I don't think I saw one).

**Comment by appurist** [comment]
I didn't really know why that `padding` style was there in the first place, or if it was necessary, so I didn't want to submit a change that removed it. But for my local testing, that's the quick work around that I made for now.

It looks like that css stuff hasn't changed since @ekryski first added the css, so it may come from an upstream css file somewhere that has since been fixed or improved.  At least it doesn't seem to be a recent change. Perhaps it has never worked on Firefox or there has been a change in Firefox since it was first committed.

I don't know enough about padding on input fields to suggest a change. I haven't tried other values (like perhaps `1.4em 1.0em` might work fine here).  At this point, I just know that one workaround is to remove the padding style from `main.login fieldset input` and `main.signup fieldset input`, those two styles I quoted above.

**Comment by daffl** [comment]
If it doesn't change the behaviour in Chrome it would be great if you could submit the fix as a PR. I did notice some weirdness in Firefox, too but never followed up on it.

/cc @ekryski

**Comment by appurist** [comment]
No problem, submitted PR #39 with the workaround I've used.

**Comment by daffl** [comment]
Closed via #39
